src/cli.js

```diff
@@ -10,14 +10,13 @@ program
   .description('starts a development server')
   .option(
     '-p, --port <port>',
-    'set the port to serve on. defaults to 1234',
-    parseInt
+    'set the port to serve on',
+    x => parseInt(x, 10),
```

Comment:
I've seen this fail without the radix: 

![image](https://user-images.githubusercontent.com/4730164/34186724-79a07df8-e4fb-11e7-8433-2df23eaa0846.png)